docker-vertica/Dockerfile

```diff
@@ -58,6 +58,8 @@ RUN set -x \
   && mkdir -p /home/dbadmin/logrotate \
   && cp -r /opt/vertica/config/logrotate /home/dbadmin/logrotate/  \
   && cp /opt/vertica/config/logrotate_base.conf /home/dbadmin/logrotate/ \
+  && mkdir -p /home/dbadmin/share \
+  && cp -r /opt/vertica/config/share/agent* /home/dbadmin/share/ \
```

Comment:
Can we add the NO_SSH_KEYS check here so we don't have to delete the file later?